langgraph/pregel/write.py

```diff
@@ -62,47 +76,69 @@ def config_specs(self) -> list[ConfigurableFieldSpec]:
         ]
 
     def _write(self, input: Any, config: RunnableConfig) -> None:
+        # split packets and entries
```

Comment:
this code used to misbehave if the same node sent multiple updates for the same channel (not possible with state graph, but possible in pregel), it would keep only the last update for each channel (per node)